Dockerfile.buster

```diff
@@ -31,7 +33,7 @@ RUN apt-get -y update && \
     # Building ImageMagick
     git clone https://github.com/ImageMagick/ImageMagick.git && \
     cd ImageMagick && git checkout ${IM_VERSION} && \
-    ./configure --without-magick-plus-plus --disable-docs --disable-static && \
+    ./configure --without-magick-plus-plus --disable-docs --disable-static --with-libtiff && \
     make && make install && \
     ldconfig /usr/local/lib && \
     apt-get remove --autoremove --purge -y gcc make cmake curl g++ yasm git autoconf pkg-config libpng-dev libjpeg62-turbo-dev libwebp-dev libde265-dev libx265-dev && \
```

Comment:
We could probably delete `libtiff-dev` here to keep image size as small as possible

Reply:
Unfortunately that's exactly what made the trick - without `libtiff-dev` imagemagick didn't show tiff support!

Reply:
Interesting, I thought that you would need *-dev dependency during build time, but not to run.

Reply:
I think I misunderstood your point, we _can_ remove _after_ compilation - will do!